interceptors/Mementifier.cfc

```diff
@@ -238,11 +238,16 @@ component{
 
             // Verify Nullness
             thisValue = isNull( thisValue ) ? (
-                structKeyExists( thisMemento.defaults, item ) ? thisMemento.defaults[ item ] : $mementifierSettings.nullDefaultValue
-            ) : thisValue;
-
+				arrayContainsNoCase( thisMemento.defaults.keyArray(), item ) ?
```

Comment:
I don't see the value of this change to `arrayContainsNoCase` vs `structKeyExists`? Can you explain?

Reply:
Ok, is this because the `thisMemento.defaults[ item ]` can be `null`?

Reply:
Right, `structKeyExists( thisMemento.defaults, item )` will return `false` even if the `item` key exists if the value is `null`.